librz/il/il_routines.c

```diff
@@ -244,3 +244,53 @@ RZ_API RZ_OWN RzILOpBool *rz_il_op_new_ne(RZ_NONNULL RzILOpPure *x, RZ_NONNULL R
 	}
 	return rz_il_op_new_bool_inv(result);
 }
+
+static inline RZ_OWN RzILOpBool *_any_fl_is_nan(RZ_NONNULL RZ_OWN RzILOpFloat *x, RZ_NONNULL RZ_OWN RzILOpFloat *y) {
+	return rz_il_op_new_bool_or(rz_il_op_new_is_nan(x), rz_il_op_new_is_nan(y));
+}
+
+static inline RZ_OWN RzILOpBool *_fl_is_less(RZ_NONNULL RzILOpFloat *x, RZ_NONNULL RzILOpFloat *y) {
+	RzILOpFloat *x_dup = rz_il_op_pure_dup(x);
+	RzILOpFloat *y_dup = rz_il_op_pure_dup(y);
+	return rz_il_op_new_forder(x_dup, y_dup);
+}
+
+RZ_API RZ_OWN RzILOpBool *rz_il_op_new_fneq(RZ_NONNULL RzILOpFloat *x, RZ_NONNULL RzILOpFloat *y) {
```

Comment:
```suggestion
RZ_API RZ_OWN RzILOpBool *rz_il_op_new_fneq(RZ_NONNULL RZ_OWN RzILOpFloat *x, RZ_NONNULL RZ_OWN RzILOpFloat *y) {
```

Also for all the others.